ddtrace/tracer/option.go

```diff
@@ -163,6 +172,20 @@ func newConfig(opts ...StartOption) *config {
 			WithGlobalTag(key, val)(c)
 		}
 	}
+	if v := os.Getenv("DD_HTTP_CLIENT_ERROR_STATUSES"); v != "" {
```

Comment:
I would suggest putting this logic into an `init` function in the `contrib/internal/httputil` package, and then using it in `TraceAndServe`. Once we do that, _all integrations_ will automatically have the option enabled. It will be a quick and effective approach.